tests/dummy/app/templates/snippets/datepicker-1.hbs

```diff
@@ -1,6 +1,6 @@
 {{#basic-dropdown as |dropdown|}}
   <input type="text"
-    id="ember-basic-dropdown-trigger-{{dropdown.uniqueId}}"
+    data-ebd-id="{{dropdown.uniqueId}}-trigger"
```

Comment:
@cibernox would you consider this to be private API? i am thinking it might be preferable to update this example to use the yielded trigger instead. if you agree I can submit a PR.